website/docs/index.md

```diff
@@ -53,6 +53,41 @@ ln -s -f ../../scripts/pre-commit .git/hooks/pre-commit
 ```
 
 
+## How to Create a Good UNIX Interface
+
+One of our goals with this project is learning how to write programs and, more importantly,
+the importance of not writing code vs. writing code. 
+
+Here are some 
+- [good rules for writing good unix programs](https://lasr.cs.ucla.edu/ficus-members/geoff/interfaces.html)
+
+A good example can be seen in notions of progress bars. The idea of progress bars has come up twice now, from 
+two different contributors, once in cp, and once in dd.
```

Comment:
of the three commands you've given as an example two use the flag `-v`, the third uses `-V`. one outputs just file names (cp), the other outputs file names and additional information based on another flag (-t), the third does something else entirely.

the point that is being made is that these are all non-standard, incompatible, and unclear, whereas with expvar you have both a clear and standard way to enable information about the internal state of the program, as well as a meaningful way to display it (via a web browser or a tool such as expvarmon). where the expvar approach really starts to shine is when one starts to push the model: for example when attempting to monitor multiple `dd` executions on multiple machines.